Makefile

```diff
@@ -83,7 +87,7 @@ clean-doc:
 	rm -rf .sphinx/.doctrees
 
 spelling: html
-	. $(VENV) ; python3 -m pyspelling -c .sphinx/spellingcheck.yaml -j $(shell nproc)
+	. $(VENV) ; python3 -m pyspelling -c .sphinx/spellingcheck.yaml
```

Comment:
This looks like a merge error - we added the `-j` recently.

Reply:
Praise: Oh, I have overlooked that in the diff, thanks!
Note: Rebased on main and resolved with 2a895a1e8348f703bb23e978614e96b1acd0c0cb.